### DISCUSSION TITLE
Actions Tab Error 404

### DISCUSSION THREAD

**Discussion by wrboyce** [community/community#25887] [Actions]
<p>I can’t seem to access the actions tab on a bunch of my repos:</p>
<p><a href="https://github.com/wrboyce/docker-utils/actions" rel="nofollow noopener">https://github.com/wrboyce/docker-utils/actions</a></p>
<p>But some do work…</p>
<p><a href="https://github.com/wrboyce/actions-sandbox/actions" rel="nofollow noopener">https://github.com/wrboyce/actions-sandbox/actions</a></p>

**Comment by wrboyce** [comment]
<p>This seems to be happening on both old and new repos, regardless of if they have HCL or Yaml workflows defined or not. Could a member of staff check my account, please?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Can you try again now?</p>

**Comment by wrboyce** [comment]
<p>That seems to have kicked it into action, thanks!</p>
